### PR TITLE
Supress request info messages

### PR DESCRIPTION
**Pull request by ChChCheeeeo**
Adding logging.getLogger("requests").setLevel(logging.WARNING) to cralwer supresses request info messages.

As read in: https://docs.python.org/2/library/logging.html#levels

### CODE REVIEW

**Comment by eleweek** [comment]
Oh, this is nice! Thank you! I think it would be better to put the code into ```main()``` function, right after ```logging.getLogger().setLevel(logging.DEBUG)```. So if some other module decides to import crawler there is no weird interaction/conflict between that module's logging settings.

**Comment by ChChCheeeeo** [comment]
Will do. Change made.

File: crawler.py [hunk 1]
```diff
@@ -46,8 +46,9 @@ def crawl(self):
                 logging.error("An error occured while crawling!")
                 logging.error(u"An error occured while crawling {}".format(current_page_url))
                 logging.exception(e)
-
+            
             next_page_url = bs.find('a', attrs={'rel' : 'next'})['href']
+
```
Comment (eleweek) [inline crawler.py]: Could you please also remove these whitespace changes? Otherwise looks good!

**Comment by ChChCheeeeo** [comment]
I think I removed all the whitespaces changes. Not sure though if I have to resubmit a new pull request.

**Comment by eleweek** [comment]
Weird, github still shows that there are extra whitespace changes. Anyway, I merged it

**Comment by eleweek** [comment]
Thank you!

**Pull request merged**